backend/grievance_service.py

```
@@ -332,13 +380,23 @@ def get_active_grievances_by_jurisdiction(self, jurisdiction_id: int, db: Sessio
             should_close = True
 
         try:
-            return db.query(Grievance).filter(
-                and_(
-                    Grievance.current_jurisdiction_id == jurisdiction_id,
-                    Grievance.status.in_([GrievanceStatus.OPEN, GrievanceStatus.IN_PROGRESS, GrievanceStatus.ESCALATED])
+            return (
+                db.query(Grievance)
+                .filter(
+                    and_(
```

Ruff (0.15.12)

[error] Undefined name `and_`

(F821)